prowler/providers/aws/services/kinesis/kinesis_service.py

```diff
@@ -0,0 +1,69 @@
+from enum import Enum
+
+from pydantic import BaseModel
+
+from prowler.lib.logger import logger
+from prowler.lib.scan_filters.scan_filters import is_resource_filtered
+from prowler.providers.aws.lib.service.service import AWSService
+
+
+class Kinesis(AWSService):
+    def __init__(self, provider):
+        # Call AWSService's __init__
+        super().__init__("kinesis", provider)
+        self.streams = {}
+        self.__threading_call__(self._list_streams)
+        self.__threading_call__(self._describe_stream, self.streams.values())
+
+    def _list_streams(self, regional_client):
+        logger.info("Kinesis - Listing Kinesis Streams...")
+        try:
+            list_streams_paginator = regional_client.get_paginator("list_streams")
+            for page in list_streams_paginator.paginate():
+                for stream in page["StreamSummaries"]:
+                    if not self.audit_resources or (
+                        is_resource_filtered(stream["StreamARN"], self.audit_resources)
+                    ):
+                        arn = stream.get("StreamARN", "")
+                        self.streams[arn] = Stream(
+                            arn=arn,
+                            name=stream["StreamName"],
+                            region=regional_client.region,
+                            status=StreamStatus(stream.get("StreamStatus", "ACTIVE")),
+                        )
+        except Exception as error:
+            logger.error(
+                f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
+            )
+
+    def _describe_stream(self, stream):
```

Comment:
```suggestion
    def _describe_stream(self, stream: Stream):
```

Reply:
Since the `Kinesis` class is defined above the `Stream` class, Python doesn’t recognize the Stream class in annotations at that point. While we could resolve this issue by importing Stream, importing a class within the same file (from itself) seems odd.